packages/p2p/src/components/orders/order-details/order-details.jsx

```diff
@@ -1,8 +1,9 @@
-import React         from 'react';
-import PropTypes     from 'prop-types';
-import { Button }    from 'deriv-components';
-import { localize }  from 'deriv-translations';
-import FooterActions from 'Components/footer-actions/footer-actions.jsx';
+import React              from 'react';
+import PropTypes          from 'prop-types';
+import { Button, Dialog } from 'deriv-components';
```

Comment:
can we have these in two lines?

Reply:
we can move into two lines where it has more than 4 arguments according to readme

Reply:
No, that's about properties in a function, import lines should always be in a new line as long as more than two object variables

Reply:
okay, ive updated accordingly